doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -2178,10 +2178,23 @@ This operation allows to create, update and/or delete several entities in a sing
 The payload is an object with two properties:
 
 + `actionType`, to specify the kind of update action to do: either APPEND, APPEND_STRICT, UPDATE,
-  DELETE.
+  DELETE, REPLACE.
 + `entities`, an array of entities, each one specified using the JSON entity representation format
   (described in the section "JSON Entity Representation").
 
+This operation is split in as many individual operations as entities in the `entities` vector, so
+the `actionType` is executed in each one of them. Depending on the `actionType` a mapping with
+regular non-batch operations can be done:
+
+* APPEND: maps to `POST /v2/entities` (if the entity does not already exist) or `POST /v2/entities/<id>/attrs`
+  (if the entity already exists).
+* APPEND_STRICT: maps to `POST /v2/entities` (if the entity does not already exist) or
+  `POST /v2/entities/<id>/attrs?options=append` (if the entity already exists).
+* UPDATE: maps to `PATCH /v2/entities/<id>/attrs`.
+* DELETE: it does not have a direct mapping, but it would be like `DELETE /v2/entities/<id>/attrs/<attrName>` on every
```

Comment:
Check also if this could map also to `DELETE /v2/entities/<id>` (i.e. entity deletion) unders some conditions (e.g. the payload for that entity doesn't have attributes).